app/infrastructure/scrapers/lms_scraper.py

```
@@ -192,9 +207,15 @@ async def scrape_notice_list(self, page_id: str) -> list[dict]:
         tasks = [self._scrape_notice(url=url) for url in urls]
 
         details = await asyncio.gather(*tasks)
-        return [{**meta, **detail} for meta, detail in zip(metadata, details)]
+        return [
+            NoticeEntity(**{**meta, **detail})
+            for meta, detail in zip(metadata, details)
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)